appintro/src/main/java/com/github/paolorotolo/appintro/AppIntroBaseFragment.kt

```diff
@@ -75,6 +78,7 @@ abstract class AppIntroBaseFragment : Fragment(), ISlideSelectionListener, ISlid
             drawable = savedInstanceState.getInt(ARG_DRAWABLE)
             title = savedInstanceState.getString(ARG_TITLE)
             description = savedInstanceState.getString(ARG_DESC)
+            bgDrawable = savedInstanceState.getInt(ARG_BG_DRAWABLE)
```

Comment:
Please move next to line `90` next to `defaultBackgroundColor = savedInstanceState.getInt(ARG_BG_COLOR)`